### ISSUE TITLE
Android demo has no effect in OPPO R9m phone

### ISSUE THREAD

**Issue by kaelsass** [google/filament#1540]
**Describe the bug**
All android demo doesn't work in OPPO R9m phone. The screen is all black.
![image](https://user-images.githubusercontent.com/11434702/63431337-37425580-c451-11e9-8428-d2ae8c7a70a6.png)
The following log is flushing:
``` 
08-21 20:21:59.493 20587-20618/com.google.android.filament.hellotriangle D/MALI: gles_state_set_error_internal:70: GLES error info:currently bound framebuffer is not valid for this operation
08-21 20:21:59.511 20587-20618/com.google.android.filament.hellotriangle D/MALI: gles_state_set_error_internal:68: GLES ctx: 0xe1e5c008, error code:0x506
08-21 20:21:59.511 20587-20618/com.google.android.filament.hellotriangle D/MALI: gles_state_set_error_internal:70: GLES error info:currently bound framebuffer is not valid for this operation
08-21 20:21:59.511 20587-20618/com.google.android.filament.hellotriangle D/MALI: gles_state_set_error_internal:68: GLES ctx: 0xe1e5c008, error code:0x506
```

**Smartphone (please complete the following information):**
 - Device: OPPO R9m
 - OS: Android 5.1

Filament version: v1.3.0, v1.3.1

**Comment by atul109** [comment]
Samsung Galaxy Grand Prime+ (SM-G532F) running on Android 6.0.1 also faced similar issue.

Logcat:
`09-02 15:04:11.366 16764-16764/? E/Zygote: v2
09-02 15:04:11.366 16764-16764/? I/libpersona: KNOX_SDCARD checking this for 10282
09-02 15:04:11.366 16764-16764/? I/libpersona: KNOX_SDCARD not a persona
09-02 15:04:11.366 16764-16764/? W/SELinux: Function: selinux_compare_spd_ram, index[1], priority [2], priority version is VE=SEPF_SECMOBILE_6.0.1_0035
09-02 15:04:11.366 16764-16764/? E/Zygote: accessInfo : 0
09-02 15:04:11.366 16764-16764/? W/SELinux: SELinux: seapp_context_lookup: seinfo=default, level=s0:c512,c768, pkgname=com.google.android.filament.gltf 
09-02 15:04:11.366 16764-16764/? I/art: Late-enabling -Xcheck:jni
09-02 15:04:11.386 16764-16764/com.google.android.filament.gltf D/TimaKeyStoreProvider: TimaSignature is unavailable
09-02 15:04:11.386 16764-16764/com.google.android.filament.gltf D/ActivityThread: Added TimaKeyStore provider
09-02 15:04:11.496 16764-16764/com.google.android.filament.gltf I/Filament: FEngine (32 bits) created at 0x9f5c0000 (threading is enabled)
09-02 15:04:11.496 16764-16781/com.google.android.filament.gltf D/Filament: FEngine resolved backend: OpenGL
09-02 15:04:11.616 16764-16764/com.google.android.filament.gltf D/ViewRootImpl: #1 mView = com.android.internal.policy.PhoneWindow$DecorView{12cccf5 I.E...... R.....ID 0,0-0,0}
09-02 15:04:11.616 16764-16792/com.google.android.filament.gltf D/OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: false
09-02 15:04:11.646 16764-16792/com.google.android.filament.gltf I/OpenGLRenderer: Initialized EGL, version 1.4
09-02 15:04:11.646 16764-16792/com.google.android.filament.gltf D/Surface: Surface::connect(this=0x9d15b000,api=1)
09-02 15:04:11.646 16764-16792/com.google.android.filament.gltf D/mali_winsys: new_window_surface returns 0x3000
09-02 15:04:11.656 16764-16764/com.google.android.filament.gltf I/gltf-bloom: RenderTarget objects are 540 x 840.
09-02 15:04:11.666 16764-16781/com.google.android.filament.gltf D/libGLESv2: DTS_GLAPI : DTS is not allowed for Package : com.google.android.filament.gltf
09-02 15:04:11.686 16764-16792/com.google.android.filament.gltf D/libGLESv1: DTS_GLAPI : DTS is not allowed for Package : com.google.android.filament.gltf
09-02 15:04:11.686 16764-16781/com.google.android.filament.gltf D/Surface: Surface::connect(this=0x9d15b700,api=1)
09-02 15:04:11.696 16764-16781/com.google.android.filament.gltf D/mali_winsys: new_window_surface returns 0x3000
09-02 15:04:11.696 16764-16781/com.google.android.filament.gltf D/MALI: gles_state_set_error_internal:63: GLES ctx: 0x9e440008, error code:0x506
09-02 15:04:11.696 16764-16781/com.google.android.filament.gltf D/MALI: gles_state_set_error_internal:64: GLES error info:currently bound framebuffer is not valid for this operation
09-02 15:04:11.706 16764-16781/com.google.android.filament.gltf D/MALI: gles_state_set_error_internal:63: GLES ctx: 0x9e440008, error code:0x506`

**Comment by bejado** [comment]
@kaelsass can you pull and retry? This may have been fixed by #1594.

**Comment by kaelsass** [comment]
@bejado It's OK now! Thanks!
